client/modules/Kanban/Kanban.css

```diff
@@ -1 +1,12 @@
 /* Kanban styles */
```

Comment:
if you don't need styles, just remove it. remember to remove also import from component file